draft-terbu-sd-jwt-vc.md

```diff
@@ -38,46 +38,58 @@ express Verifiable Credentials with JSON payload based on the securing mechanism
 
 # Introduction
 
+## The Three-Party-Model
+
 A Verifiable Credential is an tamper-evident statement made by an Issuer about
 a Subject of the Verifiable Credential. Verifiable Credentials are issued to
 Holders which can present Verifiable Credentials to Verifiers typically in form
-of Verifiable Presentations.
+of Verifiable Presentations which are secure envelops that contain Verifiable
```

Comment:
Why does a VP need to be an envelope?

Reply:
Removed envelope

Reply:
I adjusted the language. Please check if this makes more sense now.